testdata/config/tcpMetrics.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: "config.istio.io/v1alpha2"
+kind: metric
+metadata:
+  name: bytessent
+  namespace: istio-config-default
+spec:
+  value: connection.sent.bytes
+  dimensions:
```

Comment:
please see my above comment re: metrics shape.

Reply:
same comment as before.

Reply:
actually I will update this, it has an example of `istio-protocol: tcp` I don't want to remove that file.

Reply:
cool. i'll adjust the other PR as necessary when I get to it. thanks!